src/driver/sample_experimental_design.py

```diff
@@ -51,9 +51,8 @@
 ]
 
 # Here we assume all data sets specify exactly two campaigns
```

Comment:
I know that exactly two campaigns = exactly two publishers. But "all data sets specify exactly two campaigns" sounds really awkward to me... Just saying.

Reply:
Could you suggest wording that you think would be less awkward?

Reply:
Off the top of my mind.
Option 1. "all data sets include exactly two publishers". Option 2. "all data sets include exactly two publishers and specify a campaign on each publisher".
Reason: when we say "specify two campaigns", it can be the case where we have 3 pubs A, B, C but only launch campaigns on A & B.